docs/nuget-org/package-sponsorship-on-nuget-org.md

```diff
@@ -0,0 +1,120 @@
+4. If any URLs are invalid or from non-approved platforms, you'll see error messages to correct them. Otherwise, you'll see a confirmation message that your sponsorship link has been saved.
+
+   ![Screenshot of a URL from a platform that is not approved](media/sponsorship-link-error-manage-package.png)
+
+5. Each added sponsorship URL will have a **Remove** button next to it if you need to delete it.
+6. You can add up to 10 different sponsorship URLs per package ID.
+
+### Verify your sponsorship URLs display correctly
+
+1. Navigate to your package's public page on NuGet.org.
+2. Look for the **Sponsor** button in the package details **About** section.
+3. Select the **Sponsor** button to test that your URLs appear correctly in the popup.
+
+   ![Screenshot of a package details page with the sponsorship links popup open](media/sponsorship-display-links.png)
+
+## Finding and supporting packages
+
+### Identify packages that need sponsorship
+
+1. Browse to any package page on NuGet.org.
+2. Look for packages displaying a **Sponsor** button in the package details section.
+3. The **Sponsor** button indicates that the package maintainer is seeking financial support.
+
+   ![Screenshot of a package details page with a Sponsor button](media/sponsorship-button-package-details-page.png)
+
+### View available sponsorship options
+
+1. Select the **Sponsor** button on the package page.
+2. A popup window appears showing all available sponsorship links for that package.
+
+   ![Screenshot of a package details page with the sponsorship links popup open](media/sponsorship-display-links.png)
+
+### Choose your preferred sponsorship platform
+
+1. Review the available sponsorship options in the popup.
+2. Select your preferred platform to be redirected to the external sponsorship page.
+3. The link opens in a new tab or window, keeping the NuGet package page open.
+
+> [!IMPORTANT]
+> These links take you to third-party platforms. Microsoft isn't affiliated with or responsible for the content or practices of third-party platforms.
+
+## Frequently asked questions
+
+**Can I add sponsorship information to older versions of my package?**
+
+Yes! Sponsorship information is managed at the package ID level, so it automatically applies to all versions of your package, including previously published versions.
+
+**What happens if my sponsorship platform URL changes?**
+
+You can update your sponsorship URLs anytime through the package management page. Changes take effect immediately across all versions.
+
+**Can I see analytics on how many people selected my sponsorship links?**
+
+No, NuGet.org doesn't track sponsorship link selections. You'll need to check analytics on your sponsorship platform directly.
+
+**Can I add custom sponsorship platforms not on the approved list?**
+
+Currently, only the approved list of platforms is supported. This helps ensure security and legitimacy of sponsorship links.
```

Comment:
We can only say "For security reasons, only the approved list of platforms is currently supported. If you need support for additional platforms, please submit a ticket through NuGet.org."